.eslintrc

```diff
@@ -1,84 +1,15 @@
 {
   "extends": "airbnb",
   "parser": "babel-eslint",
-
```

Comment:
I'm not a fan on that copy/pasted internal config, with many disabled rules (reason were not for mjml repo). I think it's better if it fits more with airbnb which is maintained, and not a big 80+ lines bunch of ignored rules.. What dyou think? :p